src/main/java/org/kairosdb/core/tiers/MetricTiersConfiguration.java

```diff
@@ -0,0 +1,50 @@
+package org.kairosdb.core.tiers;
+
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+public class MetricTiersConfiguration {
+
+    private Set<Integer> criticalChecks = new HashSet<>();
+    private Set<Integer> importantChecks = new HashSet<>();
+    private Integer queryDistanceHoursLimit = -1;
+    private Integer queryMaxCheckTier = -1;
+
+    public Integer getQueryDistanceHoursLimit() {
+        return queryDistanceHoursLimit;
+    }
+
+    public boolean isMetricActiveForQuery(final String metricName) {
+        if (queryMaxCheckTier <= 0 || queryMaxCheckTier >= 3) {
+            return true;
+        }
+
+        final String[] split = metricName.split("\\.");
+        if (split.length != 3 || !split[0].equals("zmon") || !split[1].equals("check")) {
+            return false;
```

Comment:
this means internal kairosdb metrics can't be fetched if tier throttling is in effect. I think this and next return should be true.

Reply:
I believe that kairosdb internal metrics are the last thing we have to care about if under unexpected load

Reply:
doesn't that mean we would lose visibility into our internals then?

Reply:
I believe metrics on our internals are not considered as critical